_permission_error
errno
TrashPermissionError
PY3
